Godeps/Godeps.json

```diff
@@ -896,6 +909,10 @@
 			"Rev": "c2528b2dd8352441850638a8bb678c2ad056fd3e"
 		},
 		{
+			"ImportPath": "golang.org/x/net/http2",
```

Comment:
Yeah - I think that the new grpc version wasn't compatible with something in contrib, and I tried to write some code that will be able to import stuff from pkg/kubelet. I probably had to grab old grpc version for that. 

I'm probably going to create a separate repo in main k8s.io one and drop this PR, as I don't think it's worth trying to maintain dependencies of contrib...